### PR TITLE
chore: release/v2.0.0 version bump

### PR DESCRIPTION
**Pull request by kpark277**
Issue #, if available:

Description of changes:

Checklist:

* [ ] Have you successfully deployed to an AWS account with your changes?
* [ ] Have you written new tests for your core changes, as applicable?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by htztomic — APPROVED** [review]

File: solutions/swb-app/CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Research Service Workbench (RSW) on AWS is a backend API designed to empower AWS
```
Comment (nguyen102) [inline solutions/swb-app/CHANGELOG.md:6]: Do we need to change the date on line 4 of this file?

File: solutions/swb-reference/CHANGELOG.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ Research Service Workbench (RSW) on AWS is a backend API designed to empower AWS
 
 For additional details please refer to the implementation guide [here](https://docs.aws.amazon.com/solutions/latest/research-service-workbench-on-aws/overview.html)
 
-`rsw-reference` is the entry point to Research Service Workbench. Customers can configure and deploy RSW with the provided CDK constructs and modules. 
+`swb-reference` is the entry point to Research Service Workbench. Customers can configure and deploy RSW with the provided CDK constructs and modules.
```
Comment (nguyen102) [inline solutions/swb-reference/CHANGELOG.md:9]: We're not renaming the folder structure to `rsw`?

File: solutions/swb-reference/CHANGELOG.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ Research Service Workbench (RSW) on AWS is a backend API designed to empower AWS
```
Comment (nguyen102) [inline solutions/swb-reference/CHANGELOG.md:6]: Do we need to change the date on line 4 of this file?

**Review by nguyen102 — CHANGES_REQUESTED** [review]
Do we also need to change the date in this changelog file?
https://github.com/aws-solutions/research-service-workbench-on-aws/blob/kevpark/release-v2.0.0-version-bump/CHANGELOG.md

File: solutions/swb-reference/CHANGELOG.md (hunk 2 above)
Comment (kpark277) [reply solutions/swb-reference/CHANGELOG.md:9]: We didn't have enough time to do this refactor so we're leaving these as swb-reference and swb-app.

**Review by nguyen102 — APPROVED** [review]

**Review by htztomic — APPROVED** [review]

**Review by htztomic — APPROVED** [review]

**Pull request merged**